### PR TITLE
[HER-173] Update design of the map from the cameras page

### PR DESCRIPTION
**Pull request by ConstantinescuAndrei**
## Description
Update the design of the map based of the following links:
- [Desktop](https://xd.adobe.com/view/f1281b54-d458-42c1-b28b-7dcca2c23b31-a47b/screen/d4112a22-e142-415f-bd98-4ae1844cf3b9/?fbclid=IwAR3x9zSAwHJJIrQaGxIQvAKjAwys50dFidZfz7ZnOpH4-rzQX-W-kQO4LVo)
- [Tablet](https://xd.adobe.com/view/c08e8b21-5cdf-469b-9e66-2c500a5ec043-95dd/screen/2a4a77a7-7d58-4e0c-a4d6-e6dba32384b0/specs/?fbclid=IwAR1x4Wn89kjsO0ZivtrxqBztZvh67ubeCMJGVc8eMmivnjwBRwteIr7zP20)
- [Smartphone](https://xd.adobe.com/view/c5fc8477-17c8-4014-872e-6e6575603d84-90af/screen/863da6bd-3b24-4a0c-807d-d60878312520/specs/?fbclid=IwAR2aCw3az4LwYWh4oYOW08rfN_9r8F8xh8u98QghtsQUYmHNQoKX8QUCAt4)

## Related
[HER-173](https://nerds-interns.atlassian.net/browse/HER-173)

### CODE REVIEW

File: src/cameras/map/sidebar/content/search/hooks/use-searchbar-style.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import { makeStyles } from '@material-ui/core'
 
-const root = { margin: '5%' }
+const root = {
+  margin: '5% 0 5% 0',
+  border: '1px solid #0000001F',
```
Comment (miutamihai) [inline src/cameras/map/sidebar/content/search/hooks/use-searchbar-style.js]: Let's extract this color to the theme.

File: src/cameras/map/sidebar/header/expand/styles/mobile-styles.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { makeStyles } from '@material-ui/core'
```
Comment (miutamihai) [inline src/cameras/map/sidebar/header/expand/styles/mobile-styles.js]: image
Can't we somehow combine these two?

**Review by miutamihai — CHANGES_REQUESTED** [review]

File: src/cameras/map/sidebar/content/search/hooks/use-searchbar-style.js (hunk 1 above)
Comment (ConstantinescuAndrei) [reply src/cameras/map/sidebar/content/search/hooks/use-searchbar-style.js]: Done

File: src/cameras/map/sidebar/header/expand/styles/mobile-styles.js (hunk 2 above)
Comment (ConstantinescuAndrei) [reply src/cameras/map/sidebar/header/expand/styles/mobile-styles.js]: Done

**Review by miutamihai — APPROVED** [review]

**Pull request merged**